src/Box.hs

```diff
@@ -51,6 +49,7 @@ len l =
         Text s -> length s
         Row ls -> sum $ map len ls
         Space -> 1
+        -- Tab -> ??? 0 ? TODO
```

Comment:
I'm not quite sure what to do with this either.. Tab has a different width depending on where it is in the line, but also this function is used to figure out how much space is used before a tab, and is never actually called with `Tab`.